tests/monitors/collectd_postgresql/postgresql_test.py

```diff
@@ -16,6 +17,8 @@
 
 pytestmark = [pytest.mark.collectd, pytest.mark.postgresql, pytest.mark.monitor_with_endpoints]
 
+DIR = Path(__file__).parent
```

Comment:
These would be a lot clearer if they were called `SCRIPT_DIR` or something less generic than `DIR`.